### PR TITLE
Get top miners implementation

### PR DESCRIPTION
**Pull request by azayats-leftclick**
Get top miners implementation
[task](https://www.notion.so/leftclick/Implement-Get-Top-Miners-1d3ff36b9a5e421d8f2f57ec8be975f0)

### CODE REVIEW

File: cmd/freezer/api/swagger.json [hunk 1]
```diff
@@ -220,10 +220,6 @@
                 "userId": {
                     "type": "string",
                     "example": "did:ethr:0x4B73C58370AEfcEf86A6021afCDe5673511376B2"
-                },
-                "username": {
```
Comment (victor-oancea-leftclick-io) [inline cmd/freezer/api/swagger.json:224]: we also need username in the response

File: economy/contract.go [hunk 2]
```diff
@@ -49,7 +49,6 @@ type (
 	}
 	TopMiner struct {
 		UserID            string  `json:"userId" example:"did:ethr:0x4B73C58370AEfcEf86A6021afCDe5673511376B2"`
-		Username          string  `json:"username" example:"jdoe"`
```
[the remark this answers is not in the document]
Comment (azayats-leftclick) [reply economy/contract.go:52]: We haven't it in the user_economy DDL
Should I add it?

[the remark this answers is not in the document]
Comment (victor-oancea-leftclick-io) [reply economy/contract.go:52]: yep

[the remark this answers is not in the document]
Comment (azayats-leftclick) [reply economy/contract.go]: done

File: cmd/freezer/api/swagger.json (hunk 1 above)
Comment (azayats-leftclick) [reply cmd/freezer/api/swagger.json]: done

File: economy/get_top_miners.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+		return nil, errors.Wrap(ctx.Err(), "get top miners failed because context failed")
+	}
+
+	space := userEconomySpace()
+	params := map[string]interface{}{
+		"limit":  limit,
+		"offset": offset,
+	}
+
+	sql := fmt.Sprintf(`SELECT user_id, profile_picture_url, balance
```
Comment (victor-oancea-leftclick-io) [inline economy/get_top_miners.go:23]: add username

**Review by victor-oancea-leftclick-io — APPROVED** [review]

**Comment by victor-oancea-leftclick-io** [comment]
@kfedor @nusanovkornilov-leftclick CR pls

**Review by kfedor — APPROVED** [review]

File: economy/get_top_miners.go (hunk 3 above)
Comment (azayats-leftclick) [reply economy/get_top_miners.go]: added

**Pull request merged**